conf/application.conf

```diff
@@ -99,36 +86,15 @@ controllers {
 
 }
 
-
-# Evolutions
-# ~~~~~
-# You can disable evolutions if needed
-# evolutionplugin=disabled
-
-# Logger
-# ~~~~~
-# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
-
-# Root logger:
-logger.root=ERROR
-
-# Logger used by the framework:
-logger.play=INFO
-
 # Logger provided to your application:
 logger.application=DEBUG
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
     name = ${appName}
-    rateUnit = SECONDS
```

Comment:
Are these no longer needed?

Reply:
Configuring loggers via application.conf is deprecated and triggers a warning on startup.
logger.application is retained because it (along with logger.gov.uk) is used in variable substitution inside application-json-logger.xml.  This only works because of the way we pass application config as JVM system parameters at startup.

The various metrics values are set by default from the include of Bootstrap's backend.conf.  We only need to override name & enabled.